string_paramator: "hello world!!"
bool_paramator: true